composer.json

```diff
@@ -89,7 +89,8 @@
     },
     "conflict": {
         "doctrine/common": "<2.7",
-        "doctrine/mongodb-odm": "<2.0"
+        "doctrine/mongodb-odm": "<2.0",
+        "doctrine/mongodb-odm-bundle": "4.0.0-RC2"
```

Comment:
We shouldn't do this...

Reply:
We will forget it.

Reply:
And it only conflicts with a specific version so when RC3 is out it'll break again. :laughing:

Reply:
RC3 should resolve the issue so I don't think it will break.

Reply:
Oh, okay. If it's a bug in the bundle, then I'm kind of fine with this. We can remove this conflict when RC3 is released.

Reply:
Anyway, this commit doesn't belong here. :laughing:

Reply:
it's a cherry pick to make things green :)